model/godleyIcon.cc

```diff
@@ -280,8 +280,12 @@ namespace minsky
             string::size_type start=table.cell(r,c).find_first_not_of(" ");
             if (start!=string::npos)
               {
-                FlowCoef fc(table.cell(r,c).substr(start));                      
+                FlowCoef fc(table.cell(r,c).substr(start));                                      
                 v.init=fc.str();              
+                // set initial value of stock var to init value of flow that is defined by a parameter or a constant. for ticket 1137
+                if (auto initVar=minsky().definingVar(VariableValue::valueId(group.lock(),table.cell(r,c).substr(start))))
+                  if (initVar->inputWired() && initVar->type()==VariableType::flow)
+                    v.init=initVar->ports[1]->wires()[0]->from()->item().variableCast()->vValue()->init;              
```

Comment:
This still has the problem that the variable is not connected to another variable, hence variableCast() will retun null, and you get a null dereference error.